nicegui/elements/qtable.js

```diff
@@ -0,0 +1,75 @@
+function eval_fields(cols) {
+    // For each Field, Sort and Format field within the column definition,
+    // look for '=>' and if so, try to eval to run the function.
+
+    cols.forEach((col) => {
+        if (!col.field || !col.field.includes("=>")) return;
```

Comment:
I'm not really sure of the implication here, but I fear that if this acts on user input data, it could be used for some sort of injection. 
I have not been able to find how though. If I change a name "Bob" to something like : 
`{'name': "<script>alert('test');</script>", 'age': {'deep': 21}},`
everything crashes. That is not related to that eval though, so I'm am not sure why at the moment.